### ISSUE TITLE
Move to -> group 

### ISSUE THREAD

**Issue by barravi** [mbnuqw/sidebery#1677]
### Description

Hello! 

When right click on a tab, then move, it would be useful to have the possibility to move tabs to existing groups as well, other than windows and panes. 

E.g. right click -> move -> groups -> group list (to avoid having the menu grow up too much).

What do you think ? It might help along my journey of replacing my bookmarks altogether with just closed tabs :-D

**Comment by UncleSnail** [comment]
Unfortunately, I don't think it is possible to add new items to the native sub-menus like "Move Tab". However, it would be possible to add a new context menu for doing this, if this is a feature that the maintainers want to see implemented.

**Comment by Nionor** [comment]
For the native menu probably not but for the menu in Sidebery it should.
Maybe something like this.
![image](https://github.com/mbnuqw/sidebery/assets/36626165/5396cfd8-533b-4939-baa3-05ef73a56687)
Where you have the option to first just move as before and then list the groups that are in that panel.

**Comment by redrockvp** [comment]
I agree with that design, but throwing in my thoughts. Maybe a hybrid/combo of the two? 

https://github.com/mbnuqw/sidebery/issues/1704

**Comment by is-this-c** [comment]
Duplicate of #619.

**Comment by justcosmic1** [comment]
I have been reaching for this function since I started using Sidebery: Trying to drag and drop multiple tabs into a group hundreds of tabs away is really cumbersome; especially before I changed the default "mouse over to select".
Move to panel/Window etcetera IMHO is much LESS useful than Move to Group. In fact I thought panels must mean groups until I investigated further (I've never used multiple panels).  
Being able to select a bunch of newly created tabs and then add them to a pre-existing Group, would be a game changer!

**Comment by clownworld123** [comment]
Please ad this option to ad tabs to groups! Remarkable that this is not standard.
